src/org/opendatakit/briefcase/model/FormStatus.java

```diff
@@ -25,6 +25,7 @@
   private String statusString = "";
   private final StringBuilder statusHistory = new StringBuilder();
   private boolean isSuccessful = true;
+  private int historyMaxSize = 8096;
```

Comment:
I'm not able to figure out where this limitation comes from! Is it something you figured out empirically? Could you please add a little comment?

I hesitate to make style suggestions out here because it's the Wild West but might be good to make this a constant if you don't mind! 😊